AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_component_validation.py

```diff
@@ -7,12 +7,13 @@
 
 import azlmbr.asset as AzAsset
 import azlmbr.math as Math
-
+from typing import Any
```

Comment:
Blank line separating import methodologies.

Reply:
that space doesn't exist in my files.... 
I will move the import and resave to see if changes take.